src/pages/OpportunityDetails.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { MapPinIcon, CalendarDaysIcon, HeartIcon } from "@heroicons/react/24/outline";
+
+function OpportunityDetails() {
+  const opportunityData = {
```

Comment:
Instead of defining a completely static data object here - leverage the dummy opportunity array that to go to the correct `id` page, make a fake function to simulate fetching the right object by ID (in the `data.js` file, for now this will just be a simple array search to mock the backend), then use that data here. You'll need to make sure the dummy data array has all the correct fields here.

Reply:
Done!